src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3874,18 +3874,18 @@
      * </pre>
      *
      * <p>parses as:</p>
-     * <pre>
-     * +--LITERAL_NEW (new)
-     *     |
-     *     +--IDENT (ArrayList)
-     *     +--LPAREN (()
-     *     +--ELIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (50)
-     *     +--RPAREN ())
-     * </pre>
+     *       <pre>
```

Comment:
`<pre>` should not be indented

Reply:
How can I change my mistakes @AkMo3

Reply:
1. Change the file. 
2. Run `mvn clean verify`
3. Commit new changes. 
4. Squash your commit into one, see [this](https://davidwalsh.name/squash-commits-git) if you require help.
5. Force push changes

Reply:
Not resolved